sky/packages/sky/lib/src/rendering/block.dart

```diff
@@ -272,8 +294,12 @@ class RenderBlockViewport extends RenderBlockBase {
     markNeedsLayout();
   }
 
-  // you can set this from within the callback if necessary
   double _startOffset;
+  /// The offset at which to paint the first child
+  ///
+  /// Can be (and often is) negative.
```

Comment:
define what the offset means (move the children up? down? by this many pixels, or whatever)